File: python/Mahasiswa.py
#deklarasi kelas
class Mahasiswa:

    #private attributes
    __nama = ""
    __NIM = ""
    __prodi = ""
    __fakultas = ""

    #konstruktor
    def __init__(self, nama ="", NIM = "", prodi = "", fakultas = ""):
        self.__nama = nama
        self.__NIM = NIM
        self.__prodi = prodi
        self.__fakultas = fakultas

    #setter & getter

    #set
    def setnama(self, nama):
        self.__nama = nama

    def setNIM(self, NIM):
        self.__NIM = NIM

    def setprodi(self, prodi):
        self.__prodi = prodi

    def setfakultas(self, fakultas):
        self.__fakultas = fakultas

    #get
    def getnama(self):
        return self.__nama

    def getNIM(self):
        return self.__NIM
    
    def getprodi(self):
        return self.__prodi

    def getfakultas(self):
        return self.__fakultas

        

File: java/Mahasiswa.java
//Deklarasi kelas Mahasiswa
public class Mahasiswa{
    //Private Attributes
    private String nama;
    private String NIM;
    private String prodi;
    private String fakultas;
    
    //Konstruktor
    public Mahasiswa(){
        //konstruktor kosong
    }

    public Mahasiswa(String nama, String NIM, String prodi, String fakultas){
        //konstruktor dengan parameter
        this.nama = nama;
        this.NIM = NIM;
        this.prodi = prodi;
        this.fakultas = fakultas;
    }

    //setter getter

    //SETTER
    public void setnama(String nama){
        this.nama = nama;
    }

    public void setNIM(String NIM){
        this.NIM = NIM;
    }

    public void setprodi(String prodi){
        this.prodi = prodi;
    }

    public void setfakultas(String fakultas){
        this.fakultas = fakultas;
    }

    //GETTER
    public String getnama(){
        return this.nama;
    }

     public String getNIM(){
        return this.NIM;
    }

    public String getprodi(){
        return this.prodi;
    }

    public String getfakultas(){
        return this.fakultas;
    }

    //method
    @Override
    //method untuk mengubah hashcode menjadi string
    public String toString(){
        return  this.getnama() + " " + this.getNIM() + " "
        + this.getprodi() + " " + this.getfakultas();
    }
}